examples/access-control-migration/src/services/assign-project-instance-id.voter.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  AuthorizationContext,
+  AuthorizationMetadata,
+  AuthorizationDecision,
+} from '@loopback/authorization';
+import _ from 'lodash';
+
+// Instance level authorizer
+// This is a WORKAROUND to modify the authorization context
+// It is not used for making a decision, so just returns ABSTAIN
+export async function assignProjectInstanceId(
```

Comment:
We'll need a function to compute the resourceId value in authorization context, but in the end, it shouldn't be a `voter` function. Those should return a decision. (but I understand why you put it here in the meantime...it sets up the data that is used by casbinproviderauthorizer.)

Reply:
@emonddr Yeah I also had that concern, but after the meeting I think we agree that it's ok to contain per-process or post-process logic in the voters, see my comment in https://github.com/strongloop/loopback-next/pull/4571#issuecomment-584296913, especially the update at the bottom.